optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/io/jaxb/OffsetDateTimeJaxbAdapter.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaplanner.benchmark.impl.io.jaxb;
+
+import java.time.DateTimeException;
+import java.time.OffsetDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeFormatterBuilder;
+import java.time.temporal.ChronoField;
+
+import javax.xml.bind.annotation.adapters.XmlAdapter;
+
+public class OffsetDateTimeJaxbAdapter extends XmlAdapter<String, OffsetDateTime> {
```

Comment:
Is there any work being doing to support java.time out of the box in a new version of JAXB (maybe at Jakarta)? 
If so, it would nice to have TODO comment with an expiration condition and an url link.

Reply:
Already discussed. For the others:
- we merge these adapters (I will first doublecheck if there is anything available in the KIE group)
- I will try to push this upstream

Reply:
Can we move this Adaptor next to the Duration one and on both add a "// TODO <link to jakarta repo issue>>